### PR TITLE
🐛 Add generate workflow for full-chart-install test case

### PR DESCRIPTION
**Pull request by Danil-Grigorev**
**What this PR does / why we need it**:
Addressing outstanding task in https://github.com/kubernetes-sigs/cluster-api-operator/pull/207#discussion_r1285722309

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Review by Fedosin — APPROVED** [review]
/approve

File: test/e2e/resources/full-chart-install.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
         caBundle: Cg==
         service:
           name: capi-operator-webhook-service
-          namespace: 'capi-operator-system'
+          namespace: 'default'
```
Comment (furkatgofurov7) [inline test/e2e/resources/full-chart-install.yaml:20]: any reason why is this needed for generating full chart?

Comment (Danil-Grigorev) [reply test/e2e/resources/full-chart-install.yaml:20]: This is the default state of the chart.

**Review by furkatgofurov7 — COMMENTED** [review]
/lgtm
